docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -202,6 +202,16 @@ The following command operators are supported:
   TCP
     Not implemented ("-")
 
+.. _config_access_log_format_termination_details:
+
+%TERMINATION_DETAILS%
+  HTTP
+    Not implemented ("-").
+
+  TCP
+    Termination details provides additional information about why the connection is terminated by
```

Comment:
I'd be inclined to change the begining from
 Termination details provides
to
 Termination details may provide
and optionally include this for both TCP and HTTP.  I think if an L7 connection gets killed for L4 reasons we'll probably use this field, and the may covers the fact it may not be implemented yet.

Reply:
Ah, I see, that makes a lot sense. I changed to use the same description for both HTTP and TCP, let me if you have other thoughts and I will update it, thanks.